mailpoet/tasks/release/GitHubController.php

```diff
@@ -39,6 +39,57 @@ public function __construct(
     ]);
   }
 
+  /**
+   * @return array{login: string, count: int}
+   */
+  public function calculateReviewers(): array {
+    // load pull requests
+    $response = $this->httpClient->get('pulls', [
+      'query' => [
+        'per_page' => '100', // 100 is maximum, if we want more we need to implement paging
```

Comment:
I understand why we are pulling only 100 pages but wanted to note, that this would show only the most recent count (Review) for a user.

e.g for your account, there wasn't any count at all for the MP Free repo (which means You haven't reviewed any PR recently).

Reply:
@pavel-mailpoet Noted. 
Thank you for working on this 👏🏾